bigquery_etl/cli/query.py

```diff
@@ -59,12 +59,16 @@ def _queries_matching_name_pattern(pattern, sql_path, project_id):
     callback=is_valid_dir,
 )
 
-project_id_option = click.option(
-    "--project-id",
-    "--project_id",
-    help="GCP project ID",
-    default=None,
-)
+
+def project_id_option(default=None):
```

Comment:
This was initially used in the new `bqetl query generate`. I dropped that function, but this seems like a useful DRY addition.